schemas/IDLs/org.gel.models.report.avro/6.0.0/CommonInterpreted.avdl

```diff
@@ -53,7 +56,8 @@ protocol CommonInterpreted {
         xlinked_biallelic,
         xlinked_monoallelic,
         mitochondrial,
-        unknown
+        unknown,
+        na
```

Comment:
why do we need na?

Reply:
@javild `na` will be used for Cancer cases.

Reply:
yup, historically RD has `unknown` and cancer `na`. In any case, you could argue that na (ie: not applicable (but wait!? and not avialable....?)) may not be the same as unknown.